model/src/main/scala/org/broadinstitute/dsde/rawls/model/WorkspaceACL.scala

```diff
@@ -53,17 +54,31 @@ object WorkspaceAccessLevels {
     }
   }
 
+  def toPolicyName(v: WorkspaceAccessLevel): Option[String] = {
+    v match {
+      case ProjectOwner => Option("project-owner")
+      case Owner => Option("owner")
+      case Write => Option("writer")
+      case Read => Option("reader")
+      case _ => None
+    }
+  }
+
   def withName(s: String): WorkspaceAccessLevel = {
     s match {
-      case "PROJECT_OWNER" => ProjectOwner
-      case "OWNER" => Owner
-      case "WRITER" => Write
-      case "READER" => Read
-      case "NO ACCESS" => NoAccess
+      case accessLevel if accessLevel.equalsIgnoreCase("PROJECT_OWNER") => ProjectOwner
+      case accessLevel if accessLevel.equalsIgnoreCase("OWNER") => Owner
+      case accessLevel if accessLevel.equalsIgnoreCase("WRITER") => Write
+      case accessLevel if accessLevel.equalsIgnoreCase("READER") => Read
+      case accessLevel if accessLevel.equalsIgnoreCase("NO ACCESS") => NoAccess
```

Comment:
would be nice if this function can return `Either` if refactoring isn't too much effort